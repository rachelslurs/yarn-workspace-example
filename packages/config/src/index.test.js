const config = require('./index')

it('config to match config object keys', () => {
    expect(Object.keys(config)).toMatchSnapshot()
})
