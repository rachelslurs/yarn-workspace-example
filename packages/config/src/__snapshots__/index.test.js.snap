// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config to match config object keys 1`] = `
Array [
  "babel",
  "babelPath",
  "jestConfigBase",
  "createRequire",
  "requireFromConfig",
  "environmentVars",
  "enzyme",
  "enzymeAdapter",
  "testUtils",
]
`;
